waku/v2/node/wakunode2.nim

```diff
@@ -168,8 +170,9 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
     enrTcpPort = if extPort.isSome(): extPort
                  else: some(bindPort)
     enr = createEnr(nodeKey, enrIp, enrTcpPort, none(Port))
+    
   
-  if wsEnabled == true or wssEnabled == true:
+  if wsEnabled == true or wssEnabled:
```

Comment:
```suggestion
  if wsEnabled or wssEnabled:
```
Minor consistency :)

Reply:
fixed in 5eff5597eabec4afaca39814bcd04d751c41ed2f